packages/back-end/src/queryRunners/ExperimentResultsQueryRunner.ts

```diff
@@ -427,6 +434,16 @@ export const startExperimentResultQueries = async (
   return queries;
 };
 
+function getAdditionalQueryMetadataForExperiment(
+  experiment: ExperimentInterface,
+): AdditionalQueryMetadata {
+  return {
+    experimentOwner: experiment.owner,
+    experimentProject: experiment.project ?? "All",
```

Comment:
nit: oooh, do you think `All` projects make sense in the context of a query metadata? What about `None`?